vllm/config.py

```diff
@@ -262,6 +265,8 @@ def get_total_num_kv_heads(self) -> int:
         # NOTE: for falcon, when new_decoder_architecture is True, the
         # multi_query flag is ignored and we use n_head_kv for the number of
         # KV heads.
+        if hasattr(self.hf_text_config, "model_type") and self.hf_text_config.model_type=='deepseek_v2':
```

Comment:
Is this actually needed? It seems the function would already return the right number due to `"num_key_value_heads": 128` in https://huggingface.co/deepseek-ai/DeepSeek-V2/blob/main/config.json#L32 which will be returned by the logic below, right?